models/Income.model.js

```diff
@@ -0,0 +1,26 @@
+const { Schema, model } = require('mongoose');
+
+const incomeSchema = new Schema(
+  {
+    title: {
+      type: String,
+      required: [true, 'Title is required.'],
+      trim: true,
+    },
+    category: {
+      type: String,
+      enum: ['salary', 'other'],
```

Comment:
Will the user be able to add more categories? If not, i think more category options should be added, for example income from investments.

Reply:
You are right. I am going to fix to be possible to add.